src/frontend/csrs-portal/src/assets/oidc.json

```diff
@@ -0,0 +1,11 @@
+{
+    "postLoginRoute": "/welcomeuser",
```

Comment:
Hey @skiselev1 
Just curious/trying to visualize:
Where would the route lead to/point to?

Reply:
Thank you! You found error. it was fixed.